fbw-a32nx/docs/a320-simvars.md

```diff
@@ -441,6 +440,137 @@
     - Number
     - The ILS course tuned via the left/right RMP
 
+- A32NX_RMP_ILS_TUNED
```

Comment:
Please don't re-introduce this simvar. I'm trying to clean things up and make them work as they do in the real plane.

Reply:
Did you remove it on the master already? If so, it was reintroduced with a bad merge/rebase and I believe other things too

Reply:
Yeah, it's gone on master.

Reply:
Done